HTAN.model.csv

```diff
@@ -57,8 +57,11 @@ scATAC-seq Level 3,Processed data files containing peak information for cells,,"
+scDNA-seq Level 1,Single-cell DNA-seq,,"Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, Sequencing Batch ID, Library Layout, Nucleic Acid Source, Library Selection Method, Read Length, Library Preparation Kit Name, Library Preparation Kit Vendor, Library Preparation Kit Version, Adapter Name, Adapter Sequence, Base Caller Name, Base Caller Version, Flow Cell Barcode, Fragment Maximum Length, Fragment Mean Length, Fragment Minimum Length, Fragment Standard Deviation Length, Lane Number, Library Strand, Multiplex Barcode, Size Selection Range, Target Depth, To Trim Adapter Sequence, Adapter Content, Basic Statistics, Encoding, Kmer Content, Overrepresented Sequences, Per Base N Content, Per Base Sequence Content, Per Base Sequence Quality, Per Sequence GC Content, Per Sequence Quality Score, Per Tile Sequence Quality, Percent GC Content, Sequence Duplication Levels, Sequence Length Distribution, Total Reads, QC Workflow Type, QC Workflow Version, QC Workflow Link",,FALSE,Sequencing,Biospecimen,,
 scDNA-seq Level 2,Alignment workflows downstream of scDNA-seq Level 1,,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Parent Data File ID, Alignment Workflow Url, Alignment Workflow Type, Genomic Reference, Genomic Reference URL, Index File Name, Average Base Quality, Average Insert Size, Average Read Length, Mean Coverage, Pairs On Diff CHR, Total Reads, Proportion Reads Mapped, MapQ30, Total Uniquely Mapped, Total Unmapped reads,Proportion Reads Duplicated, Short Reads, Proportion Coverage 10X, Proportion Coverage 30X, Proportion Targets No Coverage, Proportion Base Mismatch, Proportion Mitochondrial Reads, Contamination, Contamination Error",,FALSE,Sequencing,scDNA-seq Level 1,,
+Bulk Methylation-seq Level 1,"Raw data for bulk methylation sequencing, such as FASTQs and unaligned BAMs",,"Component, Filename, File Format, HTAN Biospecimen ID, Biospecimen Preparation, Nucleic Acid Source, Bisulfite Conversion, Sequencing Platform, Replicate Type, Bulk Methylation Assay Type, Total DNA Input",,FALSE,Sequencing,Biospecimen,,

```

Comment:
Check here and elsewhere if we need to double quote where there are multiple `Parent`

Reply:
I think this is OK as it is a single parent `Sequencing` and a single DependsOn Component `Biospecimen`